atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/pool/CassandraServer.java

```diff
@@ -18,30 +18,27 @@
 
 import com.palantir.logsafe.Preconditions;
 import java.net.InetSocketAddress;
-import java.util.List;
+import java.util.Set;
 import org.immutables.value.Value;
 
 @Value.Immutable
 public interface CassandraServer {
     @Value.Parameter
-    InetSocketAddress cassandraHostAddress();
+    String cassandraHostName();
 
     /**
-     * We do maintain a list of all IPs but do not create a client pool for each one of these. As of now this list is
-     * unused.
+     * {@code cassandraHostName()} with {@code reachableProxyIps()} form one reachable Cassandra server.
+     * While we maintain set of all IPs but do not create a client pool for each one of these.
```

Comment:
this needs rewording.
```suggestion
     * We maintain set of all IPs for this server, but do not create a client pool for each one.
```